### PR TITLE
feat(ECCW-509): DDEV commands, phpcs*.xml, phpunit*.xml.

### PR DESCRIPTION
**Pull request by Polynya**
## Include a summary of what this merge request involves (*)
This is the first of a series of tickets to improve testing.

Based on files in localgov profile.
DDEV commands:
- phpcs - check PHP coding standards on custom modules and ecc_theme themes
- phpcbf - fix PHP coding standards on custom modules and ecc_theme themes
- phpunit - run PHPUnit tests on custom modules
- phpunit-lgd - run PHPUnit tests on LGD modules, profile and theme

XML files:
- phpcs-ecc.xml - ruleset for PHP coding standards on custom modules and ecc_theme themes
- phpcs-lgd.xml - ruleset for PHP coding standards on LGD modules, profile and theme
- phpunit-ecc.xml - PHPUnit configuration for testing custom modules and ecc_theme themes
- phpunit-lgd.xml - PHPUnit configuration for testing LGD modules, profile and theme

## Call out any relevant implementation decisions
- Are there any links to back up your chosen methodology?
- Why have you taken the approach?
PHPCS reports issues in ecc_theme themes but PHPCBF does not fix them. This is to ensure that the developer fixes and commits the changes to the ecc_theme repo.
  
- Any particular problem areas?
## This PR has been tested in the following browsers
- [ ] Arc
- [ ] Edge
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] IE 11 (Windows)
- [ ] iOS Chrome
- [ ] iOS Safari
- [ ] Android Chrome
- [ ] Android Firefox
- [ ] Android default

### CODE REVIEW

File: .ddev/commands/web/phpcbf
Comment (siliconmeadow) [inline .ddev/commands/web/phpcbf:1]: @Polynya - Do you suppose the `Failed to run` msg is due to the items it couldn't fix?
image

Comment (Polynya) [reply .ddev/commands/web/phpcbf:1]: Yes, I think so. The error message is a bit confusing. Maybe we need a script to check the error code

Comment (Polynya) [reply .ddev/commands/web/phpcbf:1]: The error is caught in the script and a friendly message displayed
image

Comment (siliconmeadow) [reply .ddev/commands/web/phpcbf:1]: I just tried the two phpunit commands as well and the `php-lgd` also shows the confusing message:

image

The standalone phpunit _didn't_, but that could be due to no tests were run:

image

Comment (Polynya) [reply .ddev/commands/web/phpcbf:1]: I'm going to add a test to a module in the next PR when I add testing to the pipeline.

**Review by nbrettell — APPROVED** [review]

**Pull request merged**